src/system/SystemLayerImplSelect.cpp

```diff
@@ -499,7 +592,40 @@ void LayerImplSelect::HandleTimerComplete(TimerList::Node * timer)
     mTimerList.Remove(timer);
     mTimerPool.Invoke(timer);
 }
-#endif // CHIP_SYSTEM_CONFIG_USE_DISPATCH
+#endif
```

Comment:
Why this change?

Reply:
Must be an editing accident, possibly because this is one of the cases that should become `#elif` as per discussion above, but I was still trying to keep the separate `#if` blocks.